modules/View.js

```diff
@@ -459,159 +545,101 @@
       });
     },
 
     /**
-     * Activates all child views
-     * @method activateChildViews
-     * @deprecated 0.3.x - use this.activateTrackedViews({ shared: false }); instead
-     */
-    activateChildViews: function() {
-      this.activateTrackedViews({ shared: false });
-    },
-
-    /**
      * Binds the view as a tracked view - any recursive calls like activate, deactivate, or dispose will
-     * be done to the tracked view as well.  Except dispose for shared views.
-     *
+     * be done to the tracked view as well.  Except dispose for shared views. This method defaults to register the
+     * view as a child view unless specified by options.shared.
      * @param view {View} the tracked view
      * @param [options={}] {Object}  Optional options.
-     *   @param [options.shared=false] {Boolean} The view is a shared view instead of a child view
-     *                                           (shared views are not disposed when the parent is disposed)
+     *   @param [options.shared=false] {Boolean} If true, registers view as a shared view. These are views not owned by this parent. As compared to a child view
+     *                                           which are disposed when the parent is disposed. If false, registers view as a child view which are disposed when the parent is disposed.
      * @return {View} the tracked view
      * @method registerTrackedView
      */
     registerTrackedView: function(view, options) {
+      options = options || {};
+      options.child = options.shared ? false : true;
       var trackedViewsHash = this.__getTrackedViewsHash(options);
       trackedViewsHash[view.cid] = view;
       return view;
     },
 
     /**
-     * Binds the view as a child view - any recursive calls like activate, deactivate, or dispose will
-     * be done to the child view as well.
-     * @param view {View} the child view
-     * @return {View} the child view
-     * @method registerChildView
-     * @deprecated 0.3.x - use this.registerTrackedView(view, { shared: false }); instead
-     */
-    registerChildView: function(view) {
-      return this.registerTrackedView(view, { shared: false });
-    },
-
-    /**
      * Unbinds the tracked view - no recursive calls will be made to this shared view
      * @param view {View} the shared view
-     * @param [options={}] {Object}  Optional options.
-     *   @param [options.shared=false] {Boolean} The view is a shared view instead of a child view
-     *                                           (shared views are not disposed when the parent is disposed)
      * @return {View} the tracked view
      * @method unregisterTrackedView
      */
-    unregisterTrackedView: function(view, options) {
-      var trackedViewsHash = this.__getTrackedViewsHash(options);
-      delete trackedViewsHash[view.cid];
+    unregisterTrackedView: function(view) {
+      delete this.__childViews[view.cid];
+      delete this.__sharedViews[view.cid];
       return view;
```

Comment:
i believe its mostly a trade off decision because the map is cid to views and adding another parameter would make it bulky
[2/16/16, 1:16 PM] Kent Willis (kmwillis@mit.edu): This was the alternative i thought of, and I didn't like it - especially concerning backwards compatabiltiy.
{
  cid: {
     view: myView,
     shared: true
  }
}